client-v2/src/shared/README.md

```diff
@@ -0,0 +1,90 @@
+# Shared Collections Library
+(Still waiting for a better name for this)
+
+The shared collections library is a library written using React and Redux and flow for typing.
+It makes it easier to develop tools which need to provide the ability to curate collections of content.
+
+The shared collections library provides:
+- Components which render collections
+- Reducers and actions to store articles and collections in application state.
```

Comment:
... and selectors for common selection use cases, e.g. selecting all articles in a particular group for a given collection.